packages/fx-core/templates/plugins/resource/identity/bicep/identity.template.bicep

```diff
@@ -0,0 +1,10 @@
+param managedIdentityName string
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+  name: managedIdentityName
+  location: resourceGroup().location
+}
+
+output identityName string = managedIdentity.id
```

Comment:
the 3 output name seems not aligned with the actual meaning? for example, the identityName is actually not a name

Reply:
Actually the identityName of output is the full name, meanwhile the output of identity is the short/readable name.
As the output keys align with previous context config, I don't change them.